docs/v7_breaking_changes.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Taquito version 7
+author: Roxane Letourneau
+---
+
+:::warning Breaking changes
+With this major number update, we decided to refract some parts of the Taquito code. Please note that the migration to version 7 introduces breaking changes. 
```

Comment:
"we decided to refract some parts of the Taquito code." --> "we decided to refactor or change some parts of the Taquito API."